### PR TITLE
fix admin loading

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -84,12 +84,12 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  config.action_controller.asset_host = "http://trase.earth"
-  config.relative_url_root = '/content'
+  config.action_controller.asset_host = "//staging.trase.earth"
```
Comment (tiagojsag) [inline config/environments/production.rb]: @santostiago please change this and redeploy. perhaps it's worth creating a "staging" settings for rails?

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@
   # confirmation, reset password and unlock tokens in the database.
   # Devise will use the `secret_key_base` as its `secret_key`
   # by default. You can change it below and use your own secret key.
-  # config.secret_key = 'fce9a761f356a954bbf44fa0303ba678d040da70f7eaa0c74d04d4b7f1021ba0b0db65c0eac49f21cc5567ac025229e79bb117254c9d93f16f856cf56614c1d5'
+  config.secret_key = 'fce9a761f356a954bbf44fa0303ba678d040da70f7eaa0c74d04d4b7f1021ba0b0db65c0eac49f21cc5567ac025229e79bb117254c9d93f16f856cf56614c1d5'
```
Comment (tiagojsag) [inline config/initializers/devise.rb]: let's make this an env var

File: config/environments/production.rb (hunk 1 above)
Comment (santostiago) [reply config/environments/production.rb]: ok

File: config/initializers/devise.rb (hunk 2 above)
Comment (santostiago) [reply config/initializers/devise.rb]: ok

**Pull request merged**